tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -1,10 +1,6 @@
-from typing import Any
-from typing import cast
-from typing import Dict
-from typing import List
-from typing import Mapping
-from typing import Sequence
-from typing import Union
+from __future__ import annotations
+
+from collections.abc import Sequence
```

Comment:
I guess you unintentionally changed the import statement here.

```suggestion
from typing import Sequence
```

Reply:
I believe it is recommended by [PEP584](https://peps.python.org/pep-0585/).
If it is better to separate type annotation update from this PR I'll revert the change.